lms/djangoapps/django_comment_client/base/tests.py

```diff
@@ -387,7 +388,17 @@ def setUp(self):
     def test_create_thread(self, mock_request):
         self.create_thread_helper(mock_request)
 
-    def test_create_thread_with_context(self, mock_request):
+    def test_create_thread_standalone(self, mock_request):
+        team = CourseTeamFactory.create(
+            name="A Team",
+            course_id=self.course_id,
+            topic_id='topic_id',
+            discussion_topic_id="i4x-MITx-999-course-Robot_Super_Course"
+        )
+
+        # Add the student to the team so they can post to the commentable.
+        team.add_user(self.student)
+
         # create_thread_helper verifies that extra data are passed through to the comments service
         self.create_thread_helper(mock_request, extra_data={'context': 'standalone'})
```

Comment:
Maybe there should be a separate ``expected_data`` parameter to be used by the checks, so it is clear that it isn't being passed to the